extensions/sql-database-projects/src/models/project.ts

```diff
@@ -508,18 +508,27 @@ export class Project implements ISqlProject {
 	}
 
 	public getProjectTargetVersion(): string {
-		// check for invalid DSP
-		if (this.projFileXmlDoc.getElementsByTagName(constants.DSP).length !== 1 || this.projFileXmlDoc.getElementsByTagName(constants.DSP)[0].childNodes.length !== 1) {
-			throw new Error(constants.invalidDataSchemaProvider);
+		let dsp: string | undefined;
+
+		try {
+			dsp = this.evaluateProjectPropertyValue(constants.DSP);
```

Comment:
Ideally you would use `var` in here and get rid of the `let` line above completely, but linter in this project is configured to discourage `var`. :(

Reply:
Use of `var` in JS/TS has fallen out of favor, as `let` and `const` allow us to be more explicit with the scoping and mutability.